app/controllers/groups_controller.rb

```diff
@@ -9,5 +9,9 @@ def show
     enable_embedded_shopfront
     @hide_menu = true if @shopfront_layout == 'embedded'
     @group = EnterpriseGroup.find_by_permalink(params[:id]) || EnterpriseGroup.find(params[:id])
+    @target = "_self"
+
+    return nil unless params[:embedded_shopfront]
```

Comment:
There is a much better way of assigning `@target` in Ruby.
```ruby
@target = 
  if params[:embedded_shopfront]
    "_blank"
  else
    "_self"
  end
```

That said, these values are part of HTML and belong to the UI, not the controller.